### ISSUE TITLE
Lyrics saved in Metadata Files are shown as "."

### ISSUE THREAD

**Issue by steiraa** [jacquesh/foo_openlyrics#271]
My Lyrics wont show correctly (only a .) if saved in the ID3 V3.2 field, worked before Update to foobar 2.0 32 bit

#### Steps to reproduce
1. Open Track
2. Go to added OpenLyrics Panel

#### Expected behavior
Show the correct Lyric, as saved in the Tag (like shown in MP3Tag)

#### Versions

* foobar2000 version: v2.0 [x84] (32 bit)
* OpenLyrics version: 1.6

#### Additional information
Log:
```
Opening track for playback: "D:\Benutzer\ls15\Music\I Watched Myself Sleep\VachschlaV\I Watched Myself Sleep - Yggdrasil.mp3"
[eslyric]https://azlyrics.com/lyrics/iwatchedmyselfsleep/yggdrasil.html
2359621036
Biography Server: wikipedia: I Watched Myself Sleep: name not matched at musicbrainz
INFO-OpenLyrics: Failed to read search-avoidance info: Unsupported format or corrupted file
INFO-OpenLyrics: Searching for lyrics for artist='I Watched Myself Sleep', album='VachschlaV', title='Yggdrasil'...
INFO-OpenLyrics: Searching for lyrics in tag: 'UNSYNCED LYRICS'
INFO-OpenLyrics: Found lyrics in tag: 'UNSYNCED LYRICS'
INFO-OpenLyrics: Searching for lyrics in tag: 'UNSYNCEDLYRICS'
INFO-OpenLyrics: Successfully retrieved lyrics from source: Metadata tags
INFO-OpenLyrics: Parsing lyrics text...
INFO-OpenLyrics: Loaded lyrics already form a valid UTF-8 sequence
INFO-OpenLyrics: Parsing LRC lyric text...
INFO-OpenLyrics: Lyric loading complete
INFO-OpenLyrics: Skipping lyric save. Type: 1, Local: yes, Timestamped: no, Autosave: 0
```

**Comment by lkjdsf65** [comment]
a mi me pasa lo mismo, despues de la actualizacion de Foobar 2000 v 2.0 [x64]

**Comment by jacquesh** [comment]
I made a change to fix #247 which I *think* could also fix this. Feel free to re-open this ticket if you still experience this issue with v1.8 (which I'll release soon).
